libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -827,12 +839,19 @@ - (void)oauthCoordinatorDidFetchAuthCode:(SFOAuthCoordinator *)coordinator authI
 }
 
 - (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator didBeginAuthenticationWithView:(WKWebView *)view {
-
     SFLoginViewController *loginViewController = [self createLoginViewControllerInstance:coordinator];
     loginViewController.oauthView = view;
     SFSDKAuthViewHolder *viewHolder = [SFSDKAuthViewHolder new];
     viewHolder.loginController = loginViewController;
     viewHolder.scene = coordinator.authSession.oauthRequest.scene;
+    
+    SFLoginReason reason = coordinator.authSession.oauthRequest.loginReason;
+    [SFSDKCoreLogger i:[self class] format:@"Showing auth view, login reason: %@", reason]; // TODO string value
+    NSDictionary *userInfo = @{ kSFNotificationUserInfoCredentialsKey: coordinator.credentials,
+                                kSFNotificationUserInfoAuthTypeKey: coordinator.authInfo,
+                                kSFNotificationUserInfoLoginReasonKey: @(coordinator.authSession.oauthRequest.loginReason)};
```

Comment:
This reason should also be applied to the notification posted in the function below - `- (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator didBeginAuthenticationWithSession:(ASWebAuthenticationSession *)session`